backend/modules/simulation/router.py

```
@@ -1 +1,169 @@
+    UserProgressResponse, ScenarioSceneResponse,
+    SaveMessageRequest
+)
+
+
+router = APIRouter(prefix="/api/simulation", tags=["Simulation"])
+
+
+@router.post("/start", response_model=SimulationStartResponse)
+async def start_simulation(
+    request: SimulationStartRequest,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)